### PR TITLE
Basics

### PR DESCRIPTION
**Pull request by zardra**

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @matthewjohnston4 to be potential reviewers.

File: docs/_docs/editor/basics.md [hunk 1]
```diff
@@ -45,116 +42,112 @@ By default, when you open Nuclide, the Home page appears.
...
 
-## Project and File Explorer
+## Project Explorer
 
-The project and file explorer is on the left-hand side of Nuclide. This is where you can open
-projects, navigate through your project to open files in the [editing area](#editing-area),
-create new files and folders, etc.
+The Project Explorer is on the left side of Nuclide and contains two tabs: **File Tree** and **Source Control**. This is where you can open
+projects, navigate through your project to open files in the [Editing Area](#editing-area),
+create new files and folders, view source control information, etc.
```
Comment (JoelMarcey) [inline docs/_docs/editor/basics.md]: Nit: Could probably link to the source control section we have (Mercurial) here, but we can do that in a different pull request if you want.

**Comment by JoelMarcey** [comment]
Yeah, it looks like we are going to have to get your local branch in a good state before we can land this because of the conflict... I can help you see what can be done there.

But, the content changes look good.

**Comment by facebook-github-bot-2** [comment]
@zardra updated the pull request - [view changes](https://github.com/facebook/nuclide/pull/730/files/a6cfeb7e1f38d1a327054fb11453807b5a26c97c..b7c744bb731f9ead876724ab5fa0afe381fb2957)

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are a Facebook employee, you can view this diff [on Phabricator](https://phabricator.intern.facebook.com/D3909696).
